### PR TITLE
[sympc] Changed type of nr_parties to int32 from bytes

### PR DESCRIPTION
**Pull request by hershd23**
## Description
Changed type of nr_parties to int32 from bytes. Also commented out the previous bytes->int serialization, deserialzation logic
- modified:   packages/syft/proto/lib/sympc/session.proto
- modified:   packages/syft/src/syft/lib/sympc/session_util.py
- modified:   packages/syft/src/syft/proto/lib/sympc/session_pb2.py

Solves https://github.com/OpenMined/SyMPC/issues/105

## Affected Dependencies
None

## How has this been tested?
- Ran tests on my local machine
- Tested SyMPC runs fine with this change

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

Kindly review @gmuraru @kamathhrishi

### CODE REVIEW

File: packages/syft/src/syft/lib/sympc/session_util.py [hunk 1]
```diff
@@ -24,19 +24,20 @@ def protobuf_session_serializer(session: Session) -> MPCSession_PB:
     length_rs = session.ring_size.bit_length()
     rs_bytes = session.ring_size.to_bytes((length_rs + 7) // 8, byteorder="big")
 
-    length_nr_parties = session.nr_parties.bit_length()
-    nr_parties_bytes = session.nr_parties.to_bytes(
-        (length_nr_parties + 7) // 8, byteorder="big"
-    )
+    # length_nr_parties = session.nr_parties.bit_length()
+    # nr_parties_bytes = session.nr_parties.to_bytes(
+    #     (length_nr_parties + 7) // 8, byteorder="big"
```
Comment (rasswanth-s) [inline packages/syft/src/syft/lib/sympc/session_util.py:29]: MAYBE : We could remove the comments here.

File: packages/syft/src/syft/lib/sympc/session_util.py [hunk 2]
```diff
@@ -24,19 +24,20 @@ def protobuf_session_serializer(session: Session) -> MPCSession_PB:
...
+    # )
 
     uuid = str(session.uuid)
 
     session_pb = MPCSession_PB(
         uuid=uuid,
         config=conf_proto,
         ring_size=rs_bytes,
-        nr_parties=nr_parties_bytes,
+        # nr_parties=nr_parties_bytes,
```
Comment (rasswanth-s) [inline packages/syft/src/syft/lib/sympc/session_util.py:38]: Here also.

File: packages/syft/src/syft/lib/sympc/session_util.py [hunk 3]
```diff
@@ -60,7 +61,8 @@ def protobuf_session_deserializer(proto: MPCSession_PB) -> Session:
     _conf_dict = {key: value for key, value in conf_dict.items()}
     conf = Config(**_conf_dict)
     ring_size = int.from_bytes(proto.ring_size, "big")
-    nr_parties = int.from_bytes(proto.nr_parties, "big")
+    # nr_parties = int.from_bytes(proto.nr_parties, "big")
```
Comment (rasswanth-s) [inline packages/syft/src/syft/lib/sympc/session_util.py:64]: Same here.

**Review by rasswanth-s — APPROVED** [review]
LGTM

**Comment by kamathhrishi** [comment]
Looks good to me, why does it still have WIP in the titlle?

**Comment by hershd23** [comment]
Yeah, I'll remove the comments and make another commit. Then I'll remove WIP

File: packages/syft/src/syft/lib/sympc/session_util.py (hunk 1 above)
Comment (hershd23) [reply packages/syft/src/syft/lib/sympc/session_util.py]: Done

**Comment by hershd23** [comment]
Removed comments

**Review by kamathhrishi — APPROVED** [review]
Looks good to me.

**Pull request merged**